plugins/workbench-resources/src/components/Workbench.svelte

```diff
@@ -845,6 +849,28 @@
             props={{ ...currentView.componentProps, currentView, visibleNav, navFloat, appsDirection }}
           />
         {:else}
+        {#if !lastLoc && !currentView && getLocation().path[getLocation().path.length-2]=="tracker"}
```

Comment:
Nope, the workbench should not know about tracker at all.

Reply:
Ok,Will fix that.